CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 2.20.0 - 2020-12-09
```

Comment:
No need to manage Changelog, I do it when I release a new version. The next release will bring other improvments.

Reply:
removed

Reply:
Strange, I still can see the file in the PR

Reply:
😂 it is done but not pushed :)